package.json

```diff
@@ -81,8 +81,8 @@
     "ts-jest": "^28.0.8",
     "typescript": "^4.4.4"
   },
-  "homepage": "https://github.com/pmndrs/tunnel-rat#readme",
-  "dependencies": {
-    "zustand": "^4.3.2"
-  }
+  "peerDependencies": {
+    "react": ">=18.0.0"
+  },
```

Comment:
I've just pushed a commit to add React as a peer dependency, starting with v18 since that's where `useSyncExternalStore` was introduced.